src/scripts.js

```diff
@@ -34,13 +34,26 @@ import { changeFormView, displayTrips, displayExpenses, addDestinationSearch, sh
 import Session from './Session'
 
 const session = new Session()
+const loginForm = document.querySelector('#loginForm')
 const userName = document.querySelector('#userName')
 const loginButton = document.querySelector('#login-button')
 const userMenu = document.querySelector('#userMenu')
+const password = document.querySelector('#password')
+const error = document.querySelector('#error')
+loginButton.addEventListener('click', function() {
```

Comment:
You can probably move this back down with the rest of your event listeners! Or to be consistent move the others up here- just remember you may have to refactor your event handler functions to ES5 syntax!

Reply:
Smart thinking, thank you!